internal/http/types.go

```diff
@@ -115,6 +115,21 @@ type PodErrorEvent struct {
 	Environment string           `json:"environment,omitempty"`
 	ArtifactID  string           `json:"artifactId,omitempty"`
 }
+
+type JobConditionError struct {
+	Reason          string `json:"reason,omitempty"`
+	ErrorMessage 	string `json:"message,omitempty"`
```

Comment:
Is the difference in json key and Go field intentional? I would expect them to be alligned?

Reply:
no not intentional!